app/actions/cache.ts

```diff
@@ -0,0 +1,34 @@
+import type { cacheStateType } from "../reducers/cache";
+import type { Action } from "../reducers/types";
+import { MARK_COMPLETED, RESET_CACHE, SET_CACHE } from "../constants/actionTypes";
+export function setCache(cache: cacheStateType): Action {
+  return {
+    type: SET_CACHE,
+    payload: {
+      cache
+    }
+  };
+}
+export function resetCache(): Action {
+  return {
+    type: RESET_CACHE,
+    payload: null,
+    meta: {
+      cache: true
+    }
+  };
+}
+export function markCompleted(name: string, identifier: string, // eslint-disable-next-line flowtype/no-weak-types
```

Comment:
all `eslint-disable-next-line flowtype/*` should be removed, if necessary to be replaced by equivalent typescript rules

Reply:
somewhat related: do we have any (new) typescript linting rules?

Reply:
I removed all flowtype disables and now running `npm run-script lint` there come up 28 errors but rather on config/build files so I think we don't have to replace them.